.github/pull_request_template.md

```diff
@@ -8,20 +8,20 @@ Also make sure you have read and abide by the code of conduct:
 https://github.com/lucyparsons/OpenOversight/blob/develop/CODE_OF_CONDUCT.md
 
 If this pull request is not ready for review yet, please submit it as a draft.
+
+Please write your PR name in the present imperative tense. Examples of that tense are: "Fix issue in the
+dispatcher where…", "Improve our handling of…", etc.
 -->
 ## Fixes issue
- <!-- LINK YOUR ISSUE HERE -->
-
-## Description of Changes
 
 
-## Notes for Deployment
```

Comment:
We so rarely make deployment changes. This can just be added if relevant as its more the exception than otherwise.

Reply:
I feel like it's useful to have this section to call out when an environment variable needs to be added before the deployment or the deployer should validate anything once the deployment finishes

Reply:
Updated!